src/mon/OSDMonitor.cc

```diff
@@ -2542,17 +2542,23 @@ class C_AckMarkedDown : public C_MonOp {
     MonOpRequestRef op)
     : C_MonOp(op), osdmon(osdmon) {}
 
-  void _finish(int) override {
+  void _finish(int r) override {
     MOSDMarkMeDown *m = static_cast<MOSDMarkMeDown*>(op->get_req());
+    if(-EAGAIN == r) {
```

Comment:
could you add a space after `if`?

also, could you put `-EAGAIN` at the right side of `==` ? as we are not using the Yoda conditions in this source file. or better off using `switch-case`. as we are handling multiple cases of `r` here.

Reply:
Sorry, I have revised the submission as you suggested. I will pay more attention to coding specifications.